package.json

```diff
@@ -4,8 +4,21 @@
   "description": "Maze Game",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
You can consider adding more useful scripts, especially for running your application or build processes.